Content.Shared/Backmen/Surgery/Body/SharedBodySystem.Integrity.cs

```diff
@@ -108,33 +125,24 @@ private void ApplyPartDamage(
         bool canSever,
```

Comment:
Add the TryEvadeDamage function as a guard clause for applying the IntegrityChange.

Reply:
Where specifically should I call for it?

Reply:
Github web doesnt let me suggest on the unchanged code but it should be in ApplyPartDamage right before TryChangeIntegrity like so:
 
 if (!TryEvadeDamage(partEnt.Comp.Body.Value, GetEvadeChance(targetType)))
 TryChangeIntegrity(partEnt,
              damage * partMultiplier,
              //canSever && damage.DamageDict.Contains<>(_severingDamageTypes),
              canSever,
              targetPart,
              out _);

Reply:
Apologies for the formatting warcrime, not much I can do sadge

Reply:
Oh and also idk how to find that the DamageSpecifier contains at least on of the `_severingDamageTypes` except for shitty foreach cycle... Are there any good solutions for this?

Reply:
You'll basically do a foreach under the hood anyway. But look at OnTendWoundsStep's guard clause.